File: wtplayer/src/WTJoystick.cpp
#include "basetypes.hpp"

#include "WTJoystick.hpp"
#include "WTFile.hpp"

int WTJoystick::getCount(){
  APILOG;
  return 0;
}

int WTJoystick::getButtonCount(int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getAxisFlags(int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getPOVCount(int Joystick_Number){
  APILOG;
  return 0;
}

bool WTJoystick::setNotifyEvent(bool Turn_Joystick_Events_On,
                    int Joystick_Number){
  APILOG;
  return false;
}

int WTJoystick::getButtonState(int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getPosition(int Axis_Number,
                int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getPOVState(int POV_Number,
                int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getEventJoystick(){
  APILOG;
  return 0;
}

int WTJoystick::getEventTime(){
  APILOG;
  return 0;
}

int WTJoystick::getButtonStateEvent(){
  APILOG;
  return 0;
}

int WTJoystick::getAxisStateEvent(){
  APILOG;
  return 0;
}

int WTJoystick::getPOVStateEvent(int POV){
  APILOG;
  return 0;
}

void WTJoystick::setSensitivity(int Sensitivity,
                    int Joystick_Number){
  APILOG;
}

void WTJoystick::setDeadZone(int Dead_Zone,
                 int Joystick_Number){
  APILOG;
}

void WTJoystick::setPositionRange(int Range,
                      int Joystick_Number){
  APILOG;
}

int WTJoystick::getSensitivity(int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getDeadZone(int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getPositionRange(int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getPositionEvent(int Axis_Number){
  APILOG;
  return 0;
}

bool WTJoystick::getNotifyEvent(int Joystick_Number){
  APILOG;
  return false;
}

char* WTJoystick:: getHardwareName(int Joystick_Number){
  APILOG;
  return (char*)"";
}

void WTJoystick::reset(int Reserved){
  APILOG;
}

int WTJoystick::getEffectByName(char* Effect_Name,
                    int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getEffectCount(int Joystick_Number){
  APILOG;
  return 0;
}

char* WTJoystick::getEffectName(int Effect_Number,
                    int Joystick_Number){
  APILOG;
  return (char*)"";
}

float WTJoystick::getGain(int Effect_Number,
              int Joystick_Number){
  APILOG;
  return 0;
}

int WTJoystick::getSpecialFlags(int Joystick_Number){
  APILOG;
  return 0;
}

void WTJoystick::loadEffectsFromFile(WTFile* Effects_File,
                         int Joystick_Number){
  APILOG;
}

void WTJoystick::setGain(float Gain,
             int Effect_Number,
             int Joystick_Number){
  APILOG;
}

void WTJoystick::startEffect(int Effect_Number,
                 int Joystick_Number){
  APILOG;
}

int WTJoystick::startEffectOnButton(int Effect_Number,
                        int Button_Number,
                        int Repeat_Interval,
                        int Joystick_Number){
  APILOG;
  return 0;
}

void WTJoystick::stopEffect(int Effect_Number,
                int Joystick_Number){
  APILOG;
}

void WTJoystick::stopEffectOnButton(int Effect_Number,
                        int Button_Number,
                        int Joystick_Number){
  APILOG;
}

File: wtplayer/javasrc/wildtangent/webdriver/WTJoystick.java
package wildtangent.webdriver;

public interface WTJoystick {
    public int getButtonStateEvent();

    public int getAxisStateEvent();

    public int getPositionEvent(int var1);

    public String getHardwareName(int var1);

    public String getHardwareName();

    public void setPositionRange(int var1, int var2);

    public void setPositionRange(int var1);

    public int getPositionRange(int var1);

    public int getPositionRange();

    public int getPOVState(int var1, int var2);

    public int getPOVState(int var1);

    public int getEffectCount(int var1);

    public int getEffectCount();

    public float getGain(int var1, int var2);

    public float getGain(int var1);

    public void setGain(float var1, int var2, int var3);

    public void setGain(float var1, int var2);

    public void setOnEvent(WTEventJoystickCallback var1);

    public void loadEffectsFromFile(WTFile var1, int var2);

    public void loadEffectsFromFile(WTFile var1);

    public int getAxisFlags(int var1);

    public int getAxisFlags();

    public void startEffect(int var1, int var2);

    public void startEffect(int var1);

    public int startEffectOnButton(int var1, int var2, int var3, int var4);

    public int startEffectOnButton(int var1, int var2, int var3);

    public void stopEffect(int var1, int var2);

    public int getButtonCount(int var1);

    public int getButtonCount();

    public void reset(int var1);

    public void reset();

    public void stopEffect(int var1);

    public int getPOVCount(int var1);

    public int getPOVCount();

    public int getButtonState(int var1);

    public int getButtonState();

    public int getPOVStateEvent(int var1);

    public void stopEffectOnButton(int var1, int var2, int var3);

    public void stopEffectOnButton(int var1, int var2);

    public int getEffectByName(String var1, int var2);

    public void setSensitivity(int var1, int var2);

    public void setSensitivity(int var1);

    public void setSensitivity();

    public int getSensitivity(int var1);

    public int getSensitivity();

    public void setDeadZone(int var1, int var2);

    public void setDeadZone(int var1);

    public void setDeadZone();

    public int getDeadZone(int var1);

    public int getDeadZone();

    public int getEffectByName(String var1);

    public String getEffectName(int var1, int var2);

    public String getEffectName(int var1);

    public int getPosition(int var1, int var2);

    public int getPosition(int var1);

    public int getEventJoystick();

    public int getCount();

    public boolean setNotifyEvent(boolean var1, int var2);

    public boolean setNotifyEvent(boolean var1);

    public boolean getNotifyEvent(int var1);

    public boolean getNotifyEvent();

    public int getSpecialFlags(int var1);

    public int getSpecialFlags();

    public int getEventTime();
}

